### PR TITLE
Feature/rename workspaces

### PR DESCRIPTION
**Pull request by anilnatha**
This PR addresses:

1. The need to rename our workspace nomenclature where we use `vanilla` to instead use `python`. This includes files and folder names as well.
2. Updated "basic stable" workspace title and its associated logo

I tested a build of all the workspaces in a feature branch, and the `python` (formerly `vanilla`) and all other images built successfully.

This will introduce a breaking change in that if anyone has a script that has `conda activate vanilla` will need to change to `conda activate python`

### CODE REVIEW

**Review by marjo-luc — APPROVED** [review]

File: devfiles/python/devfile/devfile.yaml
Comment (grallewellyn) [inline devfiles/python/devfile/devfile.yaml:34]: Can we remove this extra space after `python`?

**Comment by anilnatha** [comment]
@grallewellyn Yes! Thank you for catching that. I'll make the fix right now.

File: devfiles/python/devfile/devfile.yaml
Comment (anilnatha) [reply devfiles/python/devfile/devfile.yaml]: @grallewellyn Fixed!

**Review by grallewellyn — DISMISSED** [review]
Approving because I successfully build r, pangeo, and isce3 with the CI/CD then had to manually build the new python image: 'mas.dit.maap-project.org/root/maap-workspaces/jupyterlab/python:vanilla-to-python'
I tested this image and also made sure the default conda environment in the terminal was `python`
Before we merge this PR, could you please approve the other PR for updating the packages?
We will need to do some more testing once this is deployed to make sure the Get Started page and devfiles are updated

**Comment by anilnatha** [comment]
>Before we merge this PR, could you please approve the other PR for updating the packages?
Will do. I'll wait for that to be merged, before merging this one.

>We will need to do some more testing once this is deployed to make sure the Get Started page and devfiles are updated
@grallewellyn I had the same thought. 👍🏽

**Comment by grallewellyn** [comment]
Just merged the update packages branch!

**Comment by anilnatha** [comment]
@grallewellyn Glad we merged that one first! This branch now has conflicts which I'll resolve. It'll probably require another review because of our branch protection rules.

**Comment by anilnatha** [comment]
@grallewellyn With your update packages branch merged to develop, I brought those changes into my feature branch for this PR to resolve the conflicts which triggered the review again.

**Review by grallewellyn — APPROVED** [review]
Tested successfully with images 'mas.dit.maap-project.org/root/maap-workspaces/jupyterlab/[env name]:rename-workspaces'
Locally built and pushed the new 'mas.dit.maap-project.org/root/maap-workspaces/jupyterlab/python:rename-workspaces' image

**Pull request merged**
